yellowbrick/text/dispersion.py

```diff
@@ -47,20 +47,24 @@ class DispersionPlot(TextVisualizer):
     ignore_case : boolean, default: False
 	Specify whether input  will be case-sensitive.
         
+    boundary : boolean, default: False
+        Specify whether document boundaries will be drawn
+    
     kwargs : dict
         Pass any additional keyword arguments to the super class.
     
     These parameters can be influenced later on in the visualization
     process, but can and should be set as early as possible.
     """
     
-    def __init__(self, words, ax=None, color=None, ignore_case=False, **kwargs):
+    def __init__(self, words, ax=None, color=None, ignore_case=False,
+                 boundary=False, **kwargs):
         super(DispersionPlot, self).__init__(ax=ax, **kwargs)
         
         self.color = color
         self.words = words
         self.ignore_case = ignore_case
-        
+        self.set_boundaries = boundary
```

Comment:
I think it's important that the properties on the visualizer match the variables that are passed in the arg list; Scikit-Learn using metaprogramming to detect what's in the `__init__` args for `get_params` and `set_params`, which we also make use of.